warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTerms.java

```diff
@@ -198,16 +195,16 @@ private Object expandRangeForNormalizers(LiteralRange<?> range, JexlNode node) {
             
             // Avoid extra parens around the expansion
             if (1 == aliasedBounds.size()) {
-                return aliasedBounds.get(0);
+                // we assume the Optional container is not empty as we have already created a new BoundedRange from the node
+                return aliasedBounds.iterator().next();
             } else {
-                List<ASTReferenceExpression> var = JexlASTHelper.wrapInParens(aliasedBounds);
+                List<ASTReferenceExpression> var = new ArrayList(aliasedBounds);
```

Comment:
You still need the JexlASTHelper.wrapInParens() part...

Reply:
I'll look into adding a few other unit tests because fun fact: this change passed all tests before pushing.

Reply:
I think the parens are required for a few edge cases, so I could believe that.